docs/user/getting_started.md

```diff
@@ -0,0 +1,252 @@
+curl -Lo kind "https://github.com/kubernetes-sigs/kind/releases/download/$ver/kind-$(uname)-amd64"
+chmod +x kind
+mv kind /usr/local/bin
+```
+
+The recommended way to try out Mizar is with Kind.
+Kind can be used to run a multi-node Kubernetes cluster with Docker containers locally on your machine.
+You can find [instructions for installing Kind on it's official site here.](https://kind.sigs.k8s.io/docs/user/quick-start/)
+
+* Enter the mizar directory and run the ```bootstrap.sh``` script.
+    * This script will install the neccessary components to compile Mizar, and run it's unit tests. These include
+        * [Clang-7](https://clang.llvm.org) (For code compilation)
+        * [Llvm-7](https://llvm.org) (For code compilation)
+         * [Cmocka](https://cmocka.org) (For unit testing)
+        * [Build Essentials](https://packages.ubuntu.com/xenial/build-essential) (For code compilation)
+        * [Python](https://www.python.org) (For running the management plane and tests)
+        * [Docker](https://www.docker.com) (For management plane and tests)
+    * There is also an optional kernel update included in the script if you wish to update to a compatible version
+
+### Linux Kernel Update
+
+For TCP to function properly, you will need to update your Kernel version to at least 5.6-rc2 on every node. A script, ```kernel_update.sh``` is provided in the Mizar repo to download and update your machine's kernel if you do not wish to build the kernel source code yourself.
+
+
+## Setup
+
+### Compiling Mizar
+
+After completing the prerequisites above, you can now compile mizar. Enter the mizar directory and run the command below.
+
+```
+$ make
+```
+
+* You can now optionally run the unit tests and coverage reports by running the commands below.
+
+```
+$ make run_unittests
+$ make lcov
+```
+
+### New Kind Cluster
+**Note**: Before proceeding with the setup script below, please make sure you have [kind](https://kind.sigs.k8s.io/docs/user/quick-start/) and [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) installed.
+Validate your kind and kubectl installations by running:
+
+```
+$ kind --version
+
+$ kubectl version --client
+```
+
+If you are testing out Mizar with Kind, a script has been included to setup a local Kubernetes cluster, and install all of the components needed to run Mizar.
+Simply run the script below in the Mizar directory.
+
+```
+$ sudo ./kind_setup.sh
+```
+
+This script does the following:
+
+* Create a multi-node local Kubernetes cluster with Kind.
+* Build the Kind-Node, Mizar-Daemon, and Mizar-Operator docker images
+* Apply all of the Mizar [CRDs](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).
+* Deploy the Mizar [Operators](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).
+* Deploy the Mizar Daemon
+* Install the Mizar CNI Plugin
```

Comment:
Reorganize this, move "New kind cluster" to "Kind (Kubernetes in Docker)"

Reply:
moved.